src/components/ErrorMessage.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+
+const S = {
+  Container: styled.div`
+    width: 100%;
+    padding: 1rem 2rem;
+    background-color: #e99002;
+    color: #fff;
+  `,
+};
+
+const ErrorMessage = ({ errorName }) => {
+  const errorMessage = `User ${errorName} is not found.`;
+  return <S.Container>{errorMessage}</S.Container>;
+};
```

Comment:
요건 뭔가 요런식으로 써도 되지 않을까 하는 🙈
(개취에욬ㅋㅋ)
```suggestion
const ErrorMessage = ({ errorName }) => (<S.Container>User {errorName} is not found.</S.Container>);
```

Reply:
아 이거 아마 프리티어랑 린트가 제대로 동작을 안해서 이렇게 했던 것 같네요 ㅎ후ㅜ